### PR TITLE
CNV_3241 Edit CD-ROMs for VMs for 4.3

### PR DESCRIPTION
**Pull request by bgaydosrh**
Testing build....

### CODE REVIEW

**Comment by bgaydosrh** [comment]
Created new module for editing CD-ROMs for VMs, conditionalized to reuse for both add to VM and VM templates.

Before merging, this PR must be Peer Reviewed and also needs QE Review. 

Code review is optional (it was reviewed initially) from @jelkosz, @glekner or @yaacov.
 
Label Peer Review Needed and enterprise-4.3 only. A separate PR will be submtted for 4.2 as the assemblies are a bit different.

Many thanks!

Bob

**Comment by bgaydosrh** [comment]
QE has provided feedback. I will implement on January 6 and then providing peer review is done I can merge this and pick to 4.3.

**Comment by bgaydosrh** [comment]
QE changes made. This only needs peer review from @ousleyp and then we can merge and pick to 4.3.

File: modules/cnv-edit-cdrom-vm.adoc
Comment (ousleyp) [inline modules/cnv-edit-cdrom-vm.adoc]: `Edit CD-ROM` should be `*Edit CD-ROM*` like in the earlier step.

File: modules/cnv-edit-cdrom-vm.adoc
Comment (ousleyp) [inline modules/cnv-edit-cdrom-vm.adoc]: Can you be more specific about where the icon is? Is it on the left or the right?

File: modules/cnv-edit-cdrom-vm.adoc
Comment (ousleyp) [inline modules/cnv-edit-cdrom-vm.adoc]: s/completed adding/added

File: modules/cnv-edit-cdrom-vm.adoc
Comment (ousleyp) [inline modules/cnv-edit-cdrom-vm.adoc]: Clarification: Is this bullet supposed to go under "Click *Add CD-ROM*" or one of the other *Media Type* steps?

**Review by ousleyp — COMMENTED** [review]
A few comments, but overall LGTM.

File: modules/cnv-edit-cdrom-vm.adoc
Comment (bgaydosrh) [reply modules/cnv-edit-cdrom-vm.adoc]: Added "Use the following procedure to configure CD-ROMs for virtual machines."

File: modules/cnv-edit-cdrom-vm.adoc
Comment (bgaydosrh) [reply modules/cnv-edit-cdrom-vm.adoc]: Good catch. Added variable.

File: modules/cnv-edit-cdrom-vm.adoc
Comment (bgaydosrh) [reply modules/cnv-edit-cdrom-vm.adoc]: Done.

File: modules/cnv-edit-cdrom-vm.adoc
Comment (bgaydosrh) [reply modules/cnv-edit-cdrom-vm.adoc]: Changed to "Click the pencil icon to the right of the *CD-ROMs* label to open the *Edit CD-ROM* window."

File: modules/cnv-edit-cdrom-vm.adoc
Comment (bgaydosrh) [reply modules/cnv-edit-cdrom-vm.adoc]: Changed to:  When all CD-ROMs are added, click *Save*.

File: modules/cnv-edit-cdrom-vm.adoc
Comment (bgaydosrh) [reply modules/cnv-edit-cdrom-vm.adoc]: Yeah, I think it's better under the first sub-bullet as you first notice this on Windows when the Edit dialog opens. So....

.. Select the type of CD-ROM configuration in the *Media Type* field.
* On Windows systems, *Windows guest tools* is attached by default in
the *Media Type* field.

I think it's better on the first because the default Windows behavior overrides this task step. But let me know if you think the second sub-bullet is a better place for it.

**Review by bgaydosrh — COMMENTED** [review]
I must have started a review by mistake :-)

**Review by sheriff-rh — APPROVED** [review]
Looks great! Merging and picking to 4.3.

File: modules/cnv-edit-cdrom-vm.adoc
Comment (vikram-redhat) [inline modules/cnv-edit-cdrom-vm.adoc]: @bgaydosrh 
@ousleyp 
@aburdenthehand 

Shouldn't these be unset the same way they were set? Inside the conditionals? These is a very small chance that they could unset similarly named variables set by someone else.

**Pull request merged**
